spec/support/corpus.rb

```diff
@@ -10,10 +10,16 @@ module MutantSpec
   #
   # rubocop:disable MethodLength
   module Corpus
+    # Not in the docs. Number from chatting with their support.
+    # 2 processors allocated per container, 4 processes works well.
+    CIRCLE_CI_CONTAINER_PROCESSES = 4
+    TMP                           = ROOT.join('tmp').freeze
```

Comment:
My decision criteria for where to place the corpus checkouts are:

a) Persists after a failed corpus test (so I can manually target it for debugging)
b) Easily reachable for manual debugging
c) Easy to implement

`Dir.tmpdir` points to a shared resource, so we have to add code path to deal with the case the most canonical name for a checked out corpus repo is already taken. Defeating b) and c). Creating a unique name is easy via `Dir.mktmpdir` but that violates a) somewhat and also contradicts b).

So I think having the checkouts inside the repo gives us isolation, and easy addressing for manual tests for free so is the overall better choice.